### DISCUSSION TITLE
How can I achieve this feature?

### DISCUSSION THREAD

**Discussion by Hashen110** [quasarframework/quasar#12455] [General - Components / Directives / etc] [answered]
How can I achieve this feature using q-input or q-select (or any other)?

![Screenshot from 2022-02-14 01-01-33](https://user-images.githubusercontent.com/37979557/153771494-ca13001a-7e27-4d7d-8e0e-86abb918b99f.png)

**Comment by gregadro** [comment]
Is this what you want to achieve? [Chips as display value](https://quasar.dev/vue-components/select#example--chips-as-display-value)

**Reply by Hashen110** [reply]
Options aren't predefined. I want the user to make options as chips.

**Comment by metalsadman** [comment, marked as answer]
@Hashen110 
https://quasar.dev/vue-components/select#create-new-values 
https://quasar.dev/vue-components/select#using-menu-and-filtering

skim thru the docs, most examples can be mixed and matched.

**Reply by Hashen110** [reply]
Thanks :)
